### ISSUE TITLE
[Keyboard Navigation- Bot Framework Web Chat - Audio]: More option control in audio section is not accessible using keyboard.

### ISSUE THREAD

**Issue by manojs5** [microsoft/BotFramework-WebChat#5370]
### Is it an issue related to Adaptive Cards?

No.

### User Experience:
Users who rely on the keyboard will be affected if the controls are not accessible with the keyboard. Keyboard users cannot perform the action as the control doesn't get accessible with the keyboard. Keyboard users will be affected as they cannot use this functionality.

### Note:
User credentials should NOT be included in the bug.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Keyboard navigation

### What is the public URL for the website?

https://compulim.github.io/webchat-loader/

### How to reproduce the issue?

1. Open URL: https://compulim.github.io/webchat-loader/ and sign in using valid credentials.
2. Navigate to web chat page and select 4.18.1-main.20240927.ea7a875 (30/9/2024) version and then select mockBot link under present label and then select open web chat in new window link at bottom.
3. Navigate to white label experience chat page and type audio word in type your message text box.
4. Navigate to the search results.
5. Verify the more control is accessible using keyboard or not.

### What do you expect?

More option controls in audio section should be accessible using keyboard.

### What actually happened?

More option controls in audio section is not accessible using keyboard.
### Observation:
while navigating in the audio section play and volume controls are accessible using enter and up and down arrow keys, but more option control is not accessible using keyboard.
using mouse more option controls are accessible.

### Do you have any screenshots or recordings to repro the issue?

![Image](https://github.com/user-attachments/assets/d0855834-79f7-4c44-94ff-e95075a42875)
https://github.com/user-attachments/assets/f699be6c-6391-4068-89a4-1f12fc808b63

### MAS reference

MAS 2.1.1 – Keyboard

### WCAG reference

2.1.1 – Keyboard

**Comment by manojs5** [comment]
#A11yMAS;#A11ySev2;#Accessibility;#AILimited;#Benchmark;#BotFrameworkWebChat-Oct24;#HCL;#MAS2.1.1;#Win11-Edge(Chromium);

**Comment by compulim** [comment]
The audio control is part of Chrome browser, not Web Chat.

Literally, it is `<audio controls>` and we let the browser render what's suitable.
